packages/solutions/app-tools/src/analyze/constants.ts

```diff
@@ -38,8 +38,12 @@ export const FILE_SYSTEM_ROUTES_COMPONENTS_DIR = 'internal_components';
 export const NESTED_ROUTE = {
   LAYOUT_FILE: 'layout',
   LAYOUT_LOADER_FILE: 'layout.loader',
+  LAYOUT_DATA_FILE: 'layout.data',
```

Comment:
What's difference between layout.loader and layout.data?

Reply:
Layout.data is a new file convention that can contain loader and action.